sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/PartitionPublishingUtils.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.implementation;
+
+/**
+ * A util class for idempotent producer partition publishing.
+ */
+public class PartitionPublishingUtils {
+    /**
+     * Increase an int value. If the increased value is over {@link Integer#MAX_VALUE}, restart from 0.
+     * @param value The number to be incremented.
+     * @param delta The number is to be incremented by delta.
+     * @return The incremented value.
+     */
+    public static int incrementSequenceNumber(int value, int delta) {
```

Comment:
Are you actually using this anywhere, or are you just adding some flexibility "in case"?

Reply:
Yes, when I increase the internal partition state, I use this method.